common/djangoapps/enrollment/views.py

```diff
@@ -71,7 +71,12 @@ class EnrollmentUserThrottle(UserRateThrottle, ApiKeyPermissionMixIn):
     rate = '40/minute'
 
     def allow_request(self, request, view):
-        return self.has_api_key_permissions(request) or super(EnrollmentUserThrottle, self).allow_request(request, view)
+        user = request.user
+
+        if user.is_superuser or user.is_staff:
```

Comment:
I originally had done this, but thinking about it -- I really do not feel the need to rate for staff / superuser.  What if we run up against the limit again as orders increase.